### PR TITLE
Bump Docker Machine 0.5.6

### PR DESCRIPTION
**Pull request by dgageot**
and Toolbox to 1.9.1i

### CODE REVIEW

**Comment by mchiang0610** [comment]
LGTM. @jeffdm @FrenchBen

**Comment by FrenchBen** [comment]
Do you mind if we bump Kitematic to 0.9.5.1 also?

Our last release was 25 days ago and fixes have been pushed since then.

File: Dockerfile.windows [hunk 1]
```diff
@@ -17,9 +17,9 @@ RUN cd innosetup && \
...
 
 # installer components
-ENV INSTALLER_VERSION 1.9.1h
+ENV INSTALLER_VERSION 1.9.1i
 ENV DOCKER_VERSION 1.9.1
-ENV DOCKER_MACHINE_VERSION 0.5.5
+ENV DOCKER_MACHINE_VERSION 0.5.6
 ENV DOCKER_COMPOSE_VERSION 1.5.2
 ENV BOOT2DOCKER_ISO_VERSION $DOCKER_VERSION
 ENV KITEMATIC_VERSION 0.9.5
```
Comment (FrenchBen) [inline Dockerfile.windows]: can you change it to `0.9.5.1` ?

**Comment by dgageot** [comment]
@FrenchBen done!

**Comment by FrenchBen** [comment]
LGTM :+1:

**Comment by jeffdm** [comment]
LGTM

**Pull request merged**